integration-test/test-pages/broker-protection/actions/fill-form.json

```diff
@@ -16,6 +16,10 @@
         {
           "type": "$generated_phone_number$",
           "selector": "#user_phone"
+        },
+        {
+          "type": "state",
```

Comment:
Not needed for this PR, but it occurs to me that we might need some additional flexibility in the future, like `stateFull` instead of `state` for the value, depending on how the broker has their `select` defined. Just making a mental note more than anything else. 😄